common/djangoapps/student/views/dashboard.py

```diff
@@ -629,6 +630,21 @@ def student_dashboard(request):
 
     enterprise_message = get_dashboard_consent_notification(request, user, course_enrollments)
 
+    recovery_email_message = None
+
+    # Update the tag info with 'enterprise_learner' if the user belongs to an enterprise customer.
+    enterprise_learner_data = get_enterprise_learner_data(user=request.user)
+    is_secondary_email_exist = UserProfile.objects.filter(secondary_email=user.email).exists()
+    if enterprise_learner_data and is_secondary_email_feature_enabled and not is_secondary_email_exist:
```

Comment:
`is_secondary_email_feature_enabled ` is a function, you need to call it to get the correct value.
```
... and is_secondary_email_feature_enabled() ...
```